api/openapi_server/models/README.md

```diff
@@ -0,0 +1,45 @@
+# Data Model
+
+```mermaid
+classDiagram
+class host{
+   *INTEGER id NOT NULL
+   VARCHAR name NOT NULL
+}
+class housing_program{
+   *INTEGER id NOT NULL
+   VARCHAR program_name NOT NULL
+   INTEGER service_provider NOT NULL
+}
+class housing_program_service_provider{
+   *INTEGER id NOT NULL
+   VARCHAR provider_name NOT NULL
+}
+class user{
+   *INTEGER id NOT NULL
+   VARCHAR email NOT NULL
+}
+housing_program_service_provider "1" -- "0..n" housing_program
+```
+
+## Data Model Generation
+
+Our data model is autogenerated using `eralchemy2`. The `eralchemy2` tool relies on `pygraphviz` and `graphviz` which are not easily installed using pip.
+
+```pwsh
+# If you don't have the choco package manager then
+# exe installers are available https://graphviz.org/download/
+choco install graphviz
+python -m pip install --use-pep517 `
+    --config-settings="--global-option=build_ext" `
+    --config-settings="--global-option="-IC:\Program Files\Graphviz\include" `
+    --config-settings="--global-option="-LC:\Program Files\Graphviz\lib" `
+    pygraphviz
+python -m pip install eralchemy2
+```
```

Comment:
I couldnt get this to work on Windows  - tried putting it into my path and everything. Im thinking it has something to do with it being a C program. Not sure. 

I jumped into WSL2 and it installed well then. 

```bash
sudo apt install graphviz libgraphviz-dev pkg-config
python3 -m pip install pygraphviz
python3 -m pip install eralchemy2
```

then from the api directory I got it to work with:
`eralchemy2 -i "sqlite:///./homeuniteus.db" -o "HomeUniteUsDataModel.md"`

However, it printed out the old ERD. I think it's because the `alembic upgrade head` create the sqllite db with the old migration. 

Will this PR add a db migration that removes the unused tables?

Reply:
Hey @agosmou, 

Thanks for the added instructions. I agree that the windows install is a pain, so I added the linux install instructions on [a7f06ab](https://github.com/hackforla/HomeUniteUs/pull/657/commits/a7f06abe3fead77ffd45cf07568b2a397d71d68e).

I'll add a comment below to address your other concerns.